src/client.ts

```diff
@@ -127,6 +127,23 @@ export class FarosClient {
     }
   }
 
+  async entrySchema(graph = 'default'): Promise<any> {
```

Comment:
let's not have a default value for graph. a) we dont do it anywhere b) it's better to have it explicitly specified

Reply:
I don't like it either, the only reason I added it is to be consistent with `gqlSchema` in the same client.

Reply:
let's remove the default from there as well

Reply:
It can be safely removed. The only client of the library so far is passing the graph

Reply:
I removed the default from the method I added, but kept it on `gqlSchema` since it's a breaking change. It's being used with the default in at least one place: https://github.com/faros-ai/airbyte-connectors/blob/98f7e0049912db573b8fa1c5271e374b6173dd87/destinations/airbyte-faros-destination/src/destination.ts#L248.

Reply:
That's coming from the feeds sdk currently.

Reply:
Thanks @eskrm , but isnt why we use TypeScript so it could check this for us?

Reply:
It's not about checking types, it's about breaking and fixing dependent code.

Reply:
Don't worry about it, I'm working on making the necessary changes to migrate airbyte-connectors to this library. The only user of this lib is the faros-graphql source so far, so it can be safely removed. 

Thanks!

Reply:
https://github.com/faros-ai/faros-js-client/pull/51